tools/raw2h/Makefile

```diff
@@ -14,7 +14,7 @@ clean:
 	@rm -rf $(TARGET) *.exe
 
 install: all
-	@cp $(TARGET) $(PSL1GHT)/host/bin/
+	@cp $(TARGET) $(PSL1GHT)/bin/
```

Comment:
AFAICT, PSL1GHT does not currently have a `bin` directory, so you'd need to create it before you can copy stuff there...

Reply:
you're right, I wasn't checking for the path. Should be fixed now.
(I've also added a validation for the `PSLIGHT` env var.)

Reply:
just to be sure, in my setup I refer to `$(PSL1GHT)` as `/usr/local/ps3dev/`

so in `$(PSL1GHT)/bin` I also have the rest of the tools, such as `ps3load cgcomp bin2s make_self` and others.

Reply:
Actually, `ps3load` et al are installed in `$(PS3DEV)/bin`, not  `$(PSL1GHT)/bin`, so maybe this should be as well?  (I have $PS3DEV and $PSL1GHT set as different locations so I can track this stuff :)

Reply:
Alright, for some reason I always assumed that both vars were pointing to the same path.

Now it's fixed, the Makefile uses `$(PS3DEV)` as expected. 😄